internal/lib/sandbox/namespaces_test.go

```diff
@@ -495,6 +507,8 @@ func setupInfraContainerWithPid(pid int) {
 	cstate.State = specs.State{
 		Pid: pid,
 	}
+	// eat error here because callers may send invalid pids to test against
+	cstate.SetProcessInformation(pid) // nolint:errcheck
```

Comment:
```suggestion
	_ := cstate.SetProcessInformation(pid) // nolint:errcheck
```
looks a tad more explicit